### PR TITLE
Add driver_station

### PR DESCRIPTION
**Pull request by Oasixer**
# Pull Request

Add driver_station package with a blank window to frc_control as per issue #48

## Contribution Checklist

- [x] I have read the [contributing](https://github.com/uwreact/frc_control/blob/melodic-devel/CONTRIBUTING.md) guide.
- [x] I have referenced relevant issues.
- [x] I have labeled accordingly.
- [x] I have detailed my changes as much as possible.

## Change Checklist

- [ ] I have added tests.
- [x] I have added necessary documentation.
- [x] I have auto formatted using clang-format or yapf.

### CODE REVIEW

**Review by matthew-reynolds — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,6 @@
 
 ### macOS ###
 .DS_Store
+
+### Python Binary###
```
Comment (Matthew-Reynolds) [inline .gitignore]: Nitpick: There should be a space after `Binary`

Comment (matthew-reynolds) [inline .gitignore]: Nitpick: There should be a space after Binary

File: driver_station/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(driver_station)
+find_package(catkin REQUIRED)
+catkin_package()
+catkin_python_setup()
+install(PROGRAMS scripts/driver_station
```
Comment (matthew-reynolds) [inline driver_station/CMakeLists.txt:7]: Nitpick: There should be a blank line before the `install` to improve readability as we expand this file. There also should not be an extra blank line at the end of the file - Every file should end in one and exactly one newline

File: driver_station/package.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>driver_station</name>
+  <version>0.0.0</version>
+  <description>Driver station based on FRC Driver Station</description>
+
+  <maintainer email="kaelan.ms@uwreact.ca">UW React Software Team</maintainer>
```
Comment (matthew-reynolds) [inline driver_station/package.xml]: Either change the email to `maintainers@uwreact.ca` to match the name, or change the name to yours to match the email

File: driver_station/package.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+  <license>BSD-3</license>
+
+  <url type="website">http://uwreact.ca</url>
+  <url type="repository">http://github.com/frc_control/driver_station</url>
+  <url type="bugtracker">http://github.com/frc_control/driver_station/issues</url>
```
Comment (matthew-reynolds) [inline driver_station/package.xml]: Bad URL - Should be the same as all other packages:
```
  <url type="bugtracker">https://github.com/uwreact/frc_control/issues</url>
  <url type="repository">https://github.com/uwreact/frc_control</url>
```

**Review by matthew-reynolds — CHANGES_REQUESTED** [review]
Couple small nitpicks, nothing major. However please review the contributing guide and double check that the code has been run through yapf.

**Review by matthew-reynolds — APPROVED** [review]

**Pull request merged**